strategies/wlth/wlth-bid-ask-ma-spread.rain

```diff
@@ -404,19 +408,19 @@ tokenrevert-amount: get(token-output-130724),
 :call<'plottables-plot>(),
   
 amount:if(
-  stablecondition stable-amount
-  if(stablerevertcondition stablerevert-amount
-    if(tokencondition token-amount
-      if(tokenrevertcondition tokenrevert-amount max-value())
+  stablerevertcondition stablerevert-amount
+  if(tokenrevertcondition tokenrevert-amount
+    if(stablecondition stable-amount
+      if(tokencondition token-amount max-value())
     )
   )
 ),
 
 price:if(
```

Comment:
Try replacing the `price` with the following segment: 
```
mul1: mul(current-price 0.995),
mul2: mul(current-price 1.005),

price: conditions(
  stablerevertcondition mul1
  tokenrevertcondition mul2
  stablecondition mul2
  tokencondition mul1
  "default" max-value()
); 
```
- The above should solve the problem of too many elements on stack, but probably will get that error if more elements were to be present on the stack. Perhaps try introducing more sources that do some of the calcs.
- Also I'm not sure of the `price` calculation above is correct because `current-price` is only in one direction and hence `mul1` and `mul2` are also in the same direction

Reply:
I have added and pushed. That gets past the truncation error.

I have put the "string" in at the end of the conditions. if I put Max Value and the direction is stable outgoing that could still go through? however if token outgoing it wont?

is there an issue there?

Reply:
No, if you put max-value is won't go through in either case.

Reply:
why is that? it still provides a price?